src/component/FetchItems.js

```diff
@@ -7,17 +7,61 @@ import DeleteToken from './DeleteToken';
 const FetchItems = ({ token, setToken, firestore }) => {
   const [empty, setEmpty] = useState(true);
 
+  // stores the number of milliseconds elapsed since January 1, 1970
+  const todayInMs = Date.now();
```

Comment:
We're using dayjs in our code - might be something to consider here too! Super handy for dealing with date/time differences!

http://zetcode.com/javascript/dayjs/

Reply:
I tried using it at first and it was giving me some trouble. But I'll try it one more time. :)

Reply:
I think this switch will be easier to take care of once we merge our branch since you have some useful logic in calculations.js.